dashboard/src/components/Header/Header.tsx

```diff
@@ -27,7 +27,6 @@ function Header() {
     if (authenticated && clusters.currentCluster) {
       dispatch(actions.namespace.fetchNamespaces(clusters.currentCluster));
       dispatch(actions.namespace.canCreate(clusters.currentCluster));
-      dispatch(actions.kube.getResourceKinds(clusters.currentCluster));
```

Comment:
I have moved this request to the `Layout` component since the error is displayed there.